packages/react-instantsearch/src/widgets/PromptSuggestions.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from 'react';
+import { useChat, useHits } from 'react-instantsearch-core';
+
+type PromptSuggestionsUiProps = {
+  agentId: string;
+};
+
+export function PromptSuggestions({ agentId }: PromptSuggestionsUiProps) {
+  const { items } = useHits();
+  const [suggestions, setSuggestions] = useState<Array<{ title: string }>>([]);
+
+  const { messages, status, sendMessage } = useChat({ agentId });
```

Comment:
is useChat the right primitive for this? is there a different /chat and /completion endpoint?